cmd/observe/observe.go

```diff
@@ -242,6 +243,8 @@ programs attached to endpoints and devices. This includes:
 		"Translate IP addresses to logical names such as pod name, FQDN, ...",
 	)
 
+	observerCmd.Flags().BoolVarP(&node, "node", "", false, "Print node name in Compact output")
```

Comment:
nit: We already use "plain" names to define filters like `--pod`, `--namespace`, `--ip`. For consistency's sake I would rename this flag to something more explicit like `--display-node-name`.